spec/acceptance/puppetserver_upgrade_spec.rb

```diff
@@ -13,29 +13,43 @@
     raise 'At least 256MB free memory required' if memoryfree_mb < 256
   end
 
+  case ENV['BEAKER_PUPPET_COLLECTION']
+  when 'puppet7'
+    from_version = '7.0.0'
+    to_version = '7.2.0'
+  when 'puppet6'
+    from_version = '6.7.0'
+    to_version = '6.7.2'
+  when 'puppet5'
+    from_version = '5.3.6'
+    to_version = '5.3.7'
```

Comment:
Somehow this finds puppet6 on Debian builds but not on CentOS. I don't understand why yet. There is a default is set [here](https://github.com/voxpupuli/voxpupuli-acceptance/blob/f780900191c2f6385eda09cdec5383c112261e34/lib/voxpupuli/acceptance/spec_helper_acceptance.rb#L5) so perhaps it's not propagating somehow.